database/migrations/2024_03_25_121553_create_booking_room_table.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
```

Comment:
Strictly speaking, you shouldn't really have any booking stuff in story 1. However I'm not going to be mean and make you undo it all